experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -4,9 +4,11 @@ import Statistics
 import LinearAlgebra
 import ClimaAtmos as CA
 import ClimaAtmos: set_surface_albedo!
+import ClimaAtmos.Parameters as CAP
```

Comment:
these previously weren't being correctly imported but it wasn't caught because we were including files that did import these